content/Projects/python-prometheus-api-client.md

```diff
@@ -0,0 +1,15 @@
+# python-prometheus-api-client
+
+This is a python wrapper for the Prometheus API. It also includes some tools for Metric data processing using Pandas Dataframes.
+
+### Installation
+
+To install the latest release:
+
+`pip install prometheus-api-client`
+
+### Links:
+ * [Source](https://github.com/aicoe/prometheus-api-client-python)
+ * [Usage examples](https://github.com/aicoe/prometheus-api-client-python#usage)
```

Comment:
Nit: here the second word is lower case, but below, both words are capitalized. Let's pick one and stick with it